R/tostring.R

```diff
@@ -154,6 +173,19 @@ setMethod("toString", "MatrixPrintForm", function(x,
 ##     x
 ## }
 
+wrap_list <- function(txt_lst, max_width) {
+  if(all(!sapply(txt_lst, grepl, pattern = "\n")) &&
```

Comment:
I think should output a warning, because the user asked to wrap but inserted already some `\n`

Reply:
Once wrapping is in, I don't think users should be putting `\n` in there title/footer information, agreed.

The only one that can't take a vector is main title. Everything else already takes a vector so if they want manual break points they can do that.

Reply:
I see, that is why I did not need to fix a `\n` in when unlisting the output. as it just outputs a vector following the newlines. In other words, it is not important to preserve the structure of `main_title`, `subtitle`, and so on. Maybe in the future, if users desire to have spacing between those, we could modify it accordingly.